backend/copr_backend/helpers.py

```diff
@@ -659,45 +642,6 @@ def format_evr(epoch, version, release):
     return f"{version}-{release}"
 
 
-def pkg_name_evr(spec_file_path, srpm_path, logger):
-    """
-    Queries a SRPM or spec file for its name and evr (epoch:version-release)
-    """
-    try:
-        spec = Specfile(spec_file_path)
-        return spec.expanded_name, format_evr(spec.expanded_epoch,
-                                              spec.expanded_version,
-                                              spec.expanded_release)
-    except Exception:  # pylint: disable=broad-exception-caught
-        # Specfile library raises too many exception to name the in except block
-        logger.exception("Exception appeared during handling spec file: %s", spec_file_path)
-        return pkg_name_evr_from_srpm(srpm_path, logger)
-
-
-
-def pkg_name_evr_from_srpm(srpm_path, logger):
-    """
-    Queries a package for its name and evr (epoch:version-release)
-    """
-    cmd = ['rpm', '-qp', '--nosignature', '--qf',
```

Comment:
I don't know :-).
But we should cover it in tests for sure.

Reply:
That was kind of a safety belt (I still don't trust python-specfile that much, also - newly it will be run on multiple architectures).  But I won't argue too much on this :-)